openmdao/docs/getting-started/install.rst

```diff
@@ -154,26 +154,27 @@ Here are the systems which we will support:
 Mac OS X
 ++++++++
 
- * Mountain Lion (10.8.5)
-
  * Mavericks (10.9.5)
 
  * Yosemite (10.10.4)
 
+ * El Capitan (10.11.x)
+
 Linux
 +++++
 
 While we have seen successful installations using RHEL and Mint, the distribution
-that we use is Ubuntu_.  The versions of Ubuntu that we will support are:
+on which we test is Ubuntu_.  The versions of Ubuntu that we will support are:
 
 .. _Ubuntu: http://www.ubuntu.com/download
 
- * Trusty Tahr (14.04.2 LTS)
-
- * Utopic Unicorn (14.10)
+ * Xenial Xerus (16.04 LTS)
 
```

Comment:
only on a VM running here on my trashcan...and it worked.  Travis will add it soon due to it being an LTS, I could hold off on listing it until then if you'd prefer?